decancer/decancer.py

```diff
@@ -146,8 +168,6 @@ async def nick_checker(self, ctx: commands.Context, *, user: discord.Member):
                 if m_nick != new_cool_nick:
                     if new_cool_nick == "name_block":
                         new_cool_nick = await self.config.guild(ctx.guild).new_custom_nick()
-                        if new_cool_nick is None:
```

Comment:
What about the fallback to default name when one isn't placed in configs?

Reply:
The default value in config is now simp name, and users can only modify that value, not completely remove it. Since it's a default value, all servers that haven't added a fallback nick yet will get this value (i think). There shouldn't be a chance for there not to be a default value in config.